doc/nrf/app_dev/bootloaders_dfu/sysbuild_image_ids.rst

```diff
@@ -0,0 +1,107 @@
+
++----------------------+--------------------------------+-------------------------------------------------+----------------------------------------------+
+| Image                | Value (if all enabled)         | Kconfig option (sysbuild)                       | Kconfig option (application/MCUboot)         |
++======================+================================+=================================================+==============================================+
+| Application          | 0                              | ``SB_CONFIG_MCUBOOT_APPLICATION_IMAGE_NUMBER``  | ``CONFIG_MCUBOOT_APPLICATION_IMAGE_NUMBER``  |
++----------------------+--------------------------------+-------------------------------------------------+----------------------------------------------+
+| Network core         | 1                              | ``SB_CONFIG_MCUBOOT_NETWORK_CORE_IMAGE_NUMBER`` | ``CONFIG_MCUBOOT_NETWORK_CORE_IMAGE_NUMBER`` |
++----------------------+--------------------------------+-------------------------------------------------+----------------------------------------------+
+| nRF7x Wi-Fi patch    | 2                              | ``SB_CONFIG_MCUBOOT_WIFI_PATCHES_IMAGE_NUMBER`` | ``CONFIG_MCUBOOT_WIFI_PATCHES_IMAGE_NUMBER`` |
++----------------------+--------------------------------+-------------------------------------------------+----------------------------------------------+
+| QSPI XIP split image | 3                              | ``SB_CONFIG_MCUBOOT_QSPI_XIP_IMAGE_NUMBER``     | ``CONFIG_MCUBOOT_QSPI_XIP_IMAGE_NUMBER``     |
++----------------------+--------------------------------+-------------------------------------------------+----------------------------------------------+
+| MCUboot              | 4 (only set for MCUboot image) | ``SB_CONFIG_MCUBOOT_MCUBOOT_IMAGE_NUMBER``      | ``CONFIG_MCUBOOT_MCUBOOT_IMAGE_NUMBER``      |
++----------------------+--------------------------------+-------------------------------------------------+----------------------------------------------+
+
+MCUboot update package version
+******************************
+
+When MCUboot updates are enabled, the firmware embeds the |NSIB| version using the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` Kconfig option.
+However, during an update, MCUboot does not check this version; it only checks the MCUboot package version.
+
+You can set the MCUboot update package version in sysbuild with ``SB_CONFIG_SECURE_BOOT_MCUBOOT_VERSION``.
+You must increase this version number when deploying an MCUboot update.
+If the version in the update is lower than the current version, MCUboot will reject the update and not transfer it to the opposing secure boot firmware slot.
+
+Additionally, make sure to load the correct update image onto the device.
+If MCUboot is currently running from the s0 slot, then you must use the s1 update, and if it is running from the s1 slot, then you must use the s0 update.
```

Comment:
```suggestion
If MCUboot is currently running from the `s0` slot, then you must use the `s1` update, and if it is running from the `s1` slot, then you must use the `s0` update.
```

Reply:
Fixed, assuming these should have been \`\`

Reply:
Yes, correct, ``.